### DISCUSSION TITLE
Share and menu buttons trade places as discussion loads

### DISCUSSION THREAD

**Discussion by KalleOlaviNiemitalo** [community/community#17431] [Mobile]
When a discussion is loading, the top right corner of the screen has a disabled share button at the left side of a disabled menu button:

![screen shot of discussion being loaded](https://user-images.githubusercontent.com/46201428/170828060-99397dc8-7a89-4de2-a1c3-4fbe17a16fb7.png)

When the discussion finishes loading and those two buttons become enabled, the share button also moves to the right side of the menu button:

![screen shot of discussion after it was loaded](https://user-images.githubusercontent.com/46201428/170828084-14597dfb-0612-4eec-825f-46737743860f.png)

I would like the order of buttons to be the same in both situations, so that I can position my finger on one of the buttons while the discussion is loading, and touch the button as soon as it becomes enabled.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.65.0 (67507275)
Device: iPhone SE 2 (iOS 15.5)
</details>

**Comment by HenSquared** [comment]
Hi @KalleOlaviNiemitalo, 

Thanks for the feedback! I'm able to reproduce this bug. I've opened an Issue internally with our team so we can discuss fixing this bug.
